src/javascripts/ng-admin/Crud/field/maFileField.js

```diff
@@ -0,0 +1,117 @@
+/*global define*/
+
+define(function (require) {
+    'use strict';
+
+    /**
+     * Edition field for a file - a file uploader.
+     *
+     * @example <ma-file-field field="field"></ma-file-field>
+     */
+    function maFileField($upload, $timeout) {
+        return {
+            scope: {
+                'field': '&',
+                'value': '='
+            },
+            restrict: 'E',
+            link: {
+                pre: function(scope) {
+                    var uploadInformation = scope.field().config.uploadInformation;
```

Comment:
You should not access the config like that, it's an internal property. Use `scope.field().uploadInformation()` instead.